test/games/test_log_file_export.py

```diff
@@ -109,6 +109,7 @@ def pytest_generate_tests(metafunc: _pytest.python.Metafunc) -> None:
         "dread/custom_start.rdvgame",  # crazy settings
         "dread/custom_patcher_data.rdvgame",  # custom patcher data
         "dread/all_settings.rdvgame",  # all settings enabled
+        "dread/hide_all_with_nothing.rdvgame",  # Model+scan+name hidden with nothing data
```

Comment:
This test seems a but unnecessary the only difference is two lines are slightly different. The rest of the rdvgame is standard.

Reply:
this test is to ensure that nothing breaks this in the future.
Its the easiest way to test things, ideas welcome for another test.

Reply:
also unsure where you got your two lines from?
every time `"map_icon": {`is used, its different now